clconf/template/template_funcs_test.go

```diff
@@ -57,3 +58,19 @@ func TestRegexReplace(t *testing.T) {
 		}
 	}
 }
+
+func TestSort(t *testing.T) {
+	test := func(name, sortAs string, src []string, expected []string) {
+		t.Run(name, func(t *testing.T) {
+			actual, err := template.Sort(src, sortAs)
+			if err != nil {
+				t.Fatalf("Error sorting: %v", err)
+			}
+			if fmt.Sprintf("%v", actual) != fmt.Sprintf("%v", expected) {
+				t.Errorf("Actual != Expected: %v != %v", actual, expected)
+			}
+		});
+	}
+	test("string", "string", []string{"b", "A", "def", "a"}, []string{"A", "a", "b", "def"})
+	test("int", "int", []string{"10", "-1", "0", "2"}, []string{"-1", "0", "2", "10"})
+}
```

Comment:
Seems like this level of feature might benefit from a cmd level test as well....  Something that shows how a source yaml could be managed by one of these new template functions.

Reply:
I'm not sure what you mean here, can you elaborate?

Reply:
I mean something [here](https://github.com/pastdev/clconf/blob/master/cmd/root_test.go) that would look something like:

```golang
func Example_testConfigGetvAppAliases() {
	b64Data := toB64(`
---
foo:
- baz
- bar
`)
	newCmd(
		"--yaml-base64", b64Data,
		"getv",
		"/",
		"--template-string", "{{ range getksvs\"/foo\" }}{{.}}{{\"\n\"}}",
	).Execute()
	// Output:
	// bar
	// baz
}
```

and one for number, more complex combination, ...

Reply:
[Added](https://github.com/pastdev/clconf/pull/26/commits/dbf041ed5e159761b5ffaec7cd3161b7f37492b6). I also made the unencrypted tests friendlier.